kotlin/android/app/build.gradle.kts

```diff
@@ -45,9 +46,21 @@ android {
     namespace = "dev.firezone.android"
     compileSdk = 34
 
-    // Life is easier if we just match the default NDK on the Ubuntu 22.04 runners
-    // https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md#android
-    ndkVersion = "27.1.12297006"
+    // We try to build against the NDK version that is pre-installed on GitHub runners.
+    // Unfortunately, those get upgraded without a grace-period where _both_ are available and updates are rolled out over time.
```

Comment:
You would think the biggest Git forge ever, run by one of the biggest tech companies ever, would get this right.